int8 OFF=0
int8 RECORDING=1

string[] groups
int8[] statuses
